linux_os/guide/system/bootloader-grub2/non-uefi/file_owner_grub2_cfg/tests/correct.pass.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
 
-chown 0:0 /boot/grub2/grub.cfg
+if [ -e /boot/grub2 ]; then
+    chown 0:0 /boot/grub2/grub.cfg
```

Comment:
So if I accidentally create this file on Ubuntu and accidentally the rule won't pick the correct path the test scenario will pass. I think it's unlikely to happen, but shouldn't we rather create different test scenarios that would have the `# platform =` specified in the header?

Reply:
See above; I'd rather we use jinja macros eventually, but that'll have to wait for the enablement to finish on my part. Then we can pull `grub2_boot_path` from the `product.yml`.

Also, who would do that to their system? :o Poor, poor computer.

Reply:
Does this suffice? Note that as we currently audit in CaC, it wouldn't catch this anyways, so IMO, testing should be aligned with what we can currently audit.

Though, with #7260, theoretically we could handle both paths... I dunno. Up to you.